chopper.py

```diff
@@ -57,6 +71,11 @@ def parse_args() -> dict[str, Any]:
         required=False,
         default="*",
     )
+
+    ## [CG] I'd probably set the default here to 'UTF-8'; Windows people can
+    ##      ponder their poor choices. But, dunno your target audience here.
+    ##      But, I also think the default is always None? You can probably just
+    #       leave it off; idk though.
```

Comment:
I'll take out the default here since you're right, but not gonna set it to utf-8. `open()` defaults to the locale's default, which is maybe preferable to specifying utf-8.